aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/AWSCognitoAuthPlugin.java

```diff
@@ -962,9 +963,19 @@ public String getVersion() {
         return BuildConfig.VERSION_NAME;
     }
 
-    private void signOutLocally(@NonNull Action onSuccess, @NonNull Consumer<AuthException> onError) {
+    private void signOutLocally(
+            @NonNull AuthSignOutOptions options,
+            @NonNull Action onSuccess,
+            @NonNull Consumer<AuthException> onError) {
+        SignOutOptions.Builder signOutOptionsBuilder =
+                SignOutOptions.builder().signOutGlobally(false).invalidateTokens(true);
```

Comment:
Instead of setting `signOutGlobally` to false, should we be getting this from the passed in `options` variable?

Reply:
In short, no per our conversation :)

Reply:
For sign out locally (which this helper method is) it needs to be set to false with invalidateTokens set to true. The isGlobalSignOut option is examined in the main method which uses this helper method.